frame/aura/src/lib.rs

```diff
@@ -89,17 +90,8 @@ pub mod pallet {
 				assert!(current_slot < new_slot, "Slot must increase");
 				CurrentSlot::<T>::put(new_slot);
 
-				if let Some(n_authorities) = <Authorities<T>>::decode_len() {
-					let authority_index = *new_slot % n_authorities as u64;
-					if T::DisabledValidators::is_disabled(authority_index as u32) {
-						panic!(
-							"Validator with index {:?} is disabled and should not be attempting to author blocks.",
-							authority_index,
-						);
-					}
-				}
-
-				// TODO [#3398] Generate offence report for all authorities that skipped their slots.
+				// TODO [#3398] Generate offence report for all authorities that skipped their
+				// slots.
```

Comment:
We seemed to loose some code here?

Reply:
not sure what do you mean?

Reply:
I'm seeing these lines removed in the diff for frame/aura/src/lib.rs:
```
if let Some(n_authorities) = <Authorities<T>>::decode_len() {
					let authority_index = *new_slot % n_authorities as u64;
				// TODO [#3398] Generate offence report for all authorities that skipped their slots.
```
I think the branch has managed to drop some of this commit: https://github.com/paritytech/substrate/commit/471db4f2b6ef9e9871fd1e9af2a56050efb3beb6

Reply:
I think the github ui is a tad dodgy here. I had to click into the file and expand it a bit before it showed me the correct diff. (First time I've see what looks like a github ui bug.)

Reply:
yeah same, it is still showing a wrong diff to me, I'll do a local compare before merge